gradle.properties

```diff
@@ -9,11 +9,11 @@ mod_version=3.3.3
 mod_base_package=de.florianmichael
 
 # via-version
-vialoader_version=2.2.5-SNAPSHOT
-viaversion_version=4.7.0
-viabackwards_version=4.7.0
-viarewind_version=2.0.4-SNAPSHOT
-snake_yml_version=2.0
+vialoader_version=2.2.9-SNAPSHOT
+viaversion_version=4.8.0-1.20.2-pre1-SNAPSHOT
+viabackwards_version=4.8.0-1.20.2-pre1-SNAPSHOT
+viarewind_version=3.0.0-SNAPSHOT
+snake_yml_version=2.2
 
 # libs
 slf4j_version=2.0.7
```

Comment:
slf4j 2.0.9 and mixin 0.8.5 exist, although "update gradle.properties" is a bit generic tbh.

Reply:
Mixin 0.8.5 might break other mods that uses mixin. 
Shade Mixin 0.7.10 won't cause more crashes.